lib/is-database-ready.js

```diff
@@ -22,16 +22,18 @@ async function isDatabaseReady() {
     const version = await postgrator.getDatabaseVersion()
 
     console.log('Database is Ready: Version', version)
-    process.exit(0)
+    return true
   } catch (error) {
     console.error(error)
     console.log('Database is not Ready')
-    process.exit(1)
+    return false
   }
 }
 
 if (require.main === module) {
-  isDatabaseReady()
+  isDatabaseReady().then(isReady => {
+    process.exit(isReady ? 0 : 1)
```

Comment:
Just curious.. why call `process.exit()` at all? All that is needed is to set `process.exitCode = isReady ? 0 : 1` and allow the process exit normally on it's own. (sorry if being pedantic, but calling `process.exit()` in this way is just a bad practice)

Reply:
No particular reason. I was just following the example in migrate.js, but I've seen that a PR has been submitted to change that as well. I'll reflect that change here too.